option.go

```diff
@@ -108,3 +108,10 @@ func WithInstrumentAttributesGetter(instrumentAttributesGetter InstrumentAttribu
 		cfg.InstrumentAttributesGetter = instrumentAttributesGetter
 	})
 }
+
+// WithDisableSkipErrMetrics will enable or disable recording any database call that responds with ErrSkip.
+func WithDisableSkipErrMetrics(disable bool) Option {
```

Comment:
Need to change the name corresponding to the above suggestion.

Reply:
Updated as suggested